.buildkite/pipeline.yml

```diff
@@ -30,6 +30,9 @@ steps:
 
   - label: '[Browser] Lint + Test'
     key: browser-lint-test
```

Comment:
Not sure if incomplete, but we probably need to set COVERAGE=true as well?

Reply:
Whoops.  Had thought that was set at the top of the file.